beacon_node/lighthouse_network/gossipsub/src/behaviour.rs

```diff
@@ -331,6 +340,9 @@ pub struct Behaviour<D = IdentityTransform, F = AllowAllSubscriptionFilter> {
 
     /// Tracks the numbers of failed messages per peer-id.
     failed_messages: HashMap<PeerId, FailedMessages>,
+
+    /// Tracks recently sent `IWANT` messages and checks if peers respond to them.
+    gossip_promises: GossipPromises,
```

Comment:
What do you think of `PendingRequests` as the name?

Reply:
This name was originally chosen back when 1.1 scoring came out. There was behaviour score penalty for not responding to these. This used to be called breaking a "Promise". I think the specs have moved away from this naming now. 

I'm personally indifferent about the naming. Would have to change a bit of the naming in the scoring tho I'd imagine to match.